dashboard/app/models/levels/level.rb

```diff
@@ -87,6 +87,7 @@ class Level < ApplicationRecord
     rubric_performance_level_4
     mini_rubric
     encrypted
+    neighborhood_enabled
```

Comment:
similarly, I think we might want this to just go on the javalab levels, check out [javalab.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/levels/javalab.rb)